### PR TITLE
Add shu-mutou to sig-ui-leads

### PR DESCRIPTION
**Pull request by shu-mutou**
Also, fix company for floreks and maciaszczykm.

Contexts:
* https://groups.google.com/g/kubernetes-dev/c/_sGC7W4YiVc/m/QkePzg--AgAJ
* #5828

### CODE REVIEW

File: sig-list.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ When the need arises, a [new SIG can be created](sig-wg-lifecycle.md)
...
-|[UI](sig-ui/README.md)|ui|* [Sebastian Florek](https://github.com/floreks), Loodse<br>* [Marcin Maciaszczyk](https://github.com/maciaszczykm), Loodse<br>|* [Slack](https://kubernetes.slack.com/messages/sig-ui)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-ui)|* Regular SIG Meeting: [Thursdays at 09:00 PT (Pacific Time) (bi-weekly)](https://groups.google.com/forum/#!forum/kubernetes-sig-ui)<br>
+|[UI](sig-ui/README.md)|ui|* [Sebastian Florek](https://github.com/floreks), Kubermatic<br>* [Marcin Maciaszczyk](https://github.com/maciaszczykm), Kubermatic<br>* [Shu Muto](https://github.com/shu-mutou), NEC<br>|* [Slack](https://kubernetes.slack.com/messages/sig-ui)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-ui)|* Regular SIG Meeting: [Thursdays at 09:00 PT (Pacific Time) (bi-weekly)](https://groups.google.com/forum/#!forum/kubernetes-sig-ui)<br>
```
Comment (floreks) [inline sig-list.md:56]: Thanks for updating this :+1:

**Comment by floreks** [comment]
+1 from me

**Comment by maciaszczykm** [comment]
+1

**Review by mrbobbytables — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**